company-profiles/nagarro.md

```diff
@@ -0,0 +1,46 @@
+# Nagarro 
+
+## Company blurb
+
+[Nagarro](https://www.nagarro.com/en) | We excel at digital product engineering. We solve complex business challenges through agility and innovation. We call it 
+
+thinking breakthroughs. see how we have helped industry leaders embrace digitalization and accelerate technology-led innovation. We care. Caring is our superpower.
+
+itdrives us to deliver excellence to our clients.
+
+
+
+## Company size
+
+Currently 16000+ Nagarrian
+
+## Remote status
+
+We believe in freedom! With this, Nagarrian can work from wherever they want, whether it's a home office, in our offices or a hybrid model.
+
+[Careers](https://www.nagarro.com/en/careers)
+
+## Region
+
+Brazil.
```

Comment:
Readme says Worldwide. This section is for where in the world the company hires, is it only Brazil, or Worldwide?